### ISSUE TITLE
proposal() is difficult to handle with high-dim p

### ISSUE THREAD

**Issue by floswald** [floswald/MomentOpt.jl#29]
* suppose you have 10 parameters in p
* suppose they all have widely different supports
* currently the `proposal` often fails to find a new candidate. it's a MVNormal that needs to be censored into the bounds of each param dimension, centered on the currentl value of p
* particularly if the current draw is close to the boundary, that makes it very unlikely to find all draws inside the boundaries.

**Comment by floswald** [comment]
this commit 427db3e on branch `hidim` modifies the `serialNormal` function to take more than 2 params/moments. 

```
julia> o[1].chains[1].evals[1].dataMoments
DataStructures.OrderedDict{Symbol,Float64} with 10 entries:
  :mu1  => -1.0
  :mu2  => 1.0
  :mu3  => 1.11125e7
  :mu4  => 22218.1
  :mu5  => 3170.8
  :mu6  => 178.838
  :mu7  => -267.014
  :mu8  => -150.022
  :mu9  => -48.0435
  :mu10 => 2.07499
```


here the result from 1000 iterations and 10 params
![t1](https://user-images.githubusercontent.com/1021558/45700448-df2e0f80-bb6c-11e8-9e01-0dc1f66a151b.png)

histograms:

<img width="1273" alt="screenshot 2018-09-18 18 02 58" src="https://user-images.githubusercontent.com/1021558/45700534-1a304300-bb6d-11e8-938a-b08f3c653c47.png">


the problem seems to be the vastly different scale of moments.

**Comment by floswald** [comment]
closed via #30
